src/test/java/microsoft/exchange/webservices/data/EwsXmlReaderTest.java

```diff
@@ -0,0 +1,77 @@
+package microsoft.exchange.webservices.data;
```

Comment:
please add a license-header like shown [here](https://github.com/OfficeDev/ews-java-api/blob/master/src/main/java/microsoft/exchange/webservices/data/AbsoluteDateTransition.java). Plz remind the line between it and the package.